app/src/main/java/com/work/restaurant/network/api/UserApi.kt

```diff
@@ -25,6 +25,7 @@ interface UserApi {
 
     @GET("/delete.php")
     fun delete(
+        @Query("user_nickname") user_nickname: String,
```

Comment:
```suggestion
        @Query("user_nickname") userNickname: String,
```
변수명관련 convention을 지켜주세요

Reply:
385b870 수정하였습니다.